Objects/typeobject.c

```diff
@@ -302,7 +302,7 @@ type_mro_modified(PyTypeObject *type, PyObject *bases) {
        each subclass when their mro is recursively updated.
      */
     Py_ssize_t i, n;
-    int custom = (Py_TYPE(type) != &PyType_Type);
+    int custom = Py_IS_TYPE(type, &PyType_Type);
```

Comment:
This one looks wrong.

Reply:
Oh.. good catch!

Reply:
I know that you made this mistake on purpose. Just to check that I really review your PRs! :-)

Reply:
lol :)